src/Blocks/components/radios/radios.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * Template for the radios Block view.
+ *
+ * @package EightshiftForms
+ */
+
+use EightshiftForms\Helpers\Components;
+
+$manifest = Components::getManifest(__DIR__);
+
+$radiosContent = Components::checkAttr('radiosContent', $attributes, $manifest);
+$radiosName = Components::checkAttr('radiosName', $attributes, $manifest);
+$radiosIsRequired = Components::checkAttr('radiosIsRequired', $attributes, $manifest);
+
+$radiosContent = str_replace('name=""', 'name="' . $radiosName . '"', $radiosContent);
```

Comment:
Perhaps it'd be more elegant to use placeholders if possible? For instance
`$radiosContent = str_replace('{{name}}', $radiosName, $radiosContent);`

Reply:
what will this achieve, can you please provide some documentation about this. tnx